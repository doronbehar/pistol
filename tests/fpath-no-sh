fpath detected, shell is not used
