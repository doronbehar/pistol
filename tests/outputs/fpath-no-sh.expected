[38;2;166;226;46m␛[0m[38;2;248;248;242m[38;2;131;148;150m[0m[38;2;102;217;239m·[0m[38;2;102;217;239m·[0m[38;2;102;217;239m·[0m[38;2;248;248;242m1[0m[38;2;166;226;46m␛[0m[38;2;248;248;242m[0m[0m[38;2;102;217;239m·[0m[38;2;166;226;46m␛[0m[38;2;248;248;242m[38;2;248;248;242mfpath[0m[38;2;102;217;239m·[0m[38;2;248;248;242mdetected,[0m[38;2;102;217;239m·[0m[38;2;248;248;242mshell[0m[38;2;102;217;239m·[0m[38;2;248;248;242mis[0m[38;2;102;217;239m·[0m[38;2;248;248;242mnot[0m[38;2;102;217;239m·[0m[38;2;248;248;242mused[0m[38;2;166;226;46m␛[0m[38;2;248;248;242m[0m[0m[38;2;249;38;114m␊[0m
